rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
Doesn't it `"\x19Ethereum Signed Message:\n{}"` and turns `rsv` into `vrs`? Its definitely not a signature of an arbitrary hash. (return type is still indeed `Signature`)